src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -285,8 +285,18 @@ def nfs_section
 
       # @return [CWM::PagerTreeItem]
       def btrfs_section
-        page = Pages::Btrfs.new(self)
-        section_item(page, Icons::BTRFS)
+        filesystems = device_graph.btrfs_filesystems
+
+        page = Pages::BtrfsFilesystems.new(filesystems, self)
+        children = filesystems.map { |f| btrfs_item(f) }
+
+        CWM::PagerTreeItem.new(page, children: children, icon: Icons::BTRFS)
```

Comment:
Now we're using the [`#section_item`](https://github.com/yast/yast-storage-ng/blob/721ea612f96651d377936836800cc701378985df/src/lib/y2partitioner/widgets/overview.rb#L336) method.